lighthouse-cli/run.js

```diff
@@ -27,12 +27,22 @@ const _PROTOCOL_TIMEOUT_EXIT_CODE = 67;
 
 /**
  * exported for testing
- * @param {string} flags
+ * @param {string|Array<string>} flags
  * @return {Array<string>}
  */
 function parseChromeFlags(flags = '') {
-  const parsed = yargsParser(
-      flags.trim(), {configuration: {'camel-case-expansion': false, 'boolean-negation': false}});
+  const trimmedFlags = (Array.isArray(flags) ?
+      // Allow multiple --chrome-flags
+      // i.e. `lighthouse --chrome-flags="--user-agent='My Agent'" --chrome-flags="--headless"`
+      flags.join(' ') :
+      // Remove wrapping single or double quotes
+      // i.e. `lighthouse --chrome-flags="--headless --user-agent='My Agent'"
+      flags.replace(/^\s*('|")(.+)\1\s*$/, '$2')
```

Comment:
I think we need to do this replacement on each item in the array too if we want this to fully work

Reply:
I thought that would not be possible but you are right there are cases where this is required.  I updated the code and tests